core/field_image.js

```diff
@@ -114,6 +116,14 @@ Blockly.FieldImage.prototype.EDITABLE = false;
  */
 Blockly.FieldImage.prototype.isDirty_ = false;
 
+/**
+ * Vertical padding below the image, which is included in the reported height of
+ * the field.
+ * @type {number}
+ * @private
+ */
+Blockly.FieldImage.Y_PADDING = 1;
```

Comment:
Should constants be at the top of the file?

Reply:
We seem to have standardized on 
- constructor
- fromJson
- properties on the class
- properties on the prototype

I'm moving this above the properties on the prototype to match.  I think the goal was to keep fromJson as close to the constructor it calls as possible, but I'm okay with changing that for all fields at the same time.

Reply:
Yeah, that order seems right. This just needs to be above the prototype declarations.

Reply:
Done.